_includes/header.html

```diff
@@ -83,12 +86,12 @@ <h1 class="title">
             </li>
           {% endfor %}
         </ul>
-      </div>
+      </nav>
 
-      <div class="flex flex-row ai-ctr rel d:hide">
+      <nav class="flex flex-row ai-ctr rel d:hide">
         <button class="flex" id="navBtn" role="button">
           <img
-            alt=""
+            alt="Click to render sub-page navigation"
```

Comment:
so, modals and popups like this are very problematic for screen readers and this doesn't actually solve the problem, but it does help a tiny bit and passes the audit

ideally we'd want to focus the screen reader onto the newly opened modal, and at the same time hide everything else until it's closed, and then return the user back to where they were

which is obviously a huge pain on a small jekyll site - so im gonna punt that for now

Reply:
The other fix, can be to remove this modal and convert it to a separate page altogether, but this adds Alt Text in the meanwhile